src/sysv/systemd/sssd-sudo.service.in

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=SSSD Sudo Service responder
+Documentation=man:sssd.conf(5)
+After=sssd.service
+BindsTo=sssd.service
+
+[Install]
+Also=sssd-sudo.socket
+
+[Service]
+ExecStartPre=-/bin/chown @SSSD_USER@:@SSSD_USER@ @logpath@/sssd_sudo.log
+ExecStart=@libexecdir@/sssd/sssd_sudo --debug-to-files --unprivileged-start
```

Comment:
Please don't send new patches now, I got distracted today and I'm still working through the code, we can do all the changes (if more are needed) at once..